src/migrations/20190829131034-create-profile.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Profiles', {
```

Comment:
Why do we have `Profile` table where we have the `Users` table? please remove it and add `department_id` as a filed in the Users table.